fileio.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
```

Comment:
does the squaring and outputting to the file, but it prints a ridiculous number of 47.61s at the end.